setup.py

```diff
@@ -164,7 +161,7 @@ def enable_log_to_stdout(logname):
     tests_require=['pytest', 'tox', 'flake8'],
     license='MIT',
     zip_safe=False,
-    keywords='kolibri',
+    keywords=['education', 'offline', 'kolibri'],
```

Comment:
I'm wondering if we will probably want to add more keywords here. But the PR looks good to me!